paperscraper/utils.py

```diff
@@ -110,17 +102,22 @@ async def _request(self, *args, **kwargs) -> aiohttp.ClientResponse:
         return response
 
 
-def check_pdf(path, verbose: Union[bool, Logger] = False) -> bool:  # noqa: FA100
+def check_pdf(path: str, verbose: Union[bool, Logger] = False) -> bool:  # noqa: FA100
     if not os.path.exists(path):
         return False
+
     try:
-        pdf = pypdf.PdfReader(path)  # noqa: F841
-    except (pypdf.errors.PyPdfError, ValueError) as e:
+        # Open the PDF file using fitz
+        with fitz.open(path) as _:
```

Comment:
```suggestion
        with fitz.open(path):
```

Why do `as _`? Seems extra to me

Reply:
fixed